invenio/utils/remote_debugger/__init__.py

```diff
@@ -21,6 +21,13 @@
 that makes remote debugging possible and easy.
 """
 
+import warnings
+
+from invenio.utils.deprecation import RemovedInInvenio21Warning
+
+warnings.warn("Remote debugger is going to be removed. "
+              "Please use native Werkzeug debugger.",
+              RemovedInInvenio21Warning)
```

Comment:
> Should we amend the deprecation policy to allow for these sort of fast track deprecations?

Yes! Especially for features that hasn't well tested and are not part of the *core* functionality.